auth/src/main/java/com/firebase/ui/auth/ui/ImeHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.firebase.ui.auth.ui;
+
+import android.view.KeyEvent;
+import android.view.inputmethod.EditorInfo;
+import android.widget.EditText;
+import android.widget.TextView;
+
+public class ImeHelper {
+    public static void addImeOnDoneListener(EditText doneEditText, final Runnable doneAction) {
```

Comment:
I'd prefer to make up an interface here (`OnDonePressedListener`) or similar rather than using the generic `Runnable` since it makes usage clearer.  Even though Runnable is a compatible interface for what we want to do, if we used it everywhere we needed a no-args function we'd have a hard time managing.